data/json/npcs/NPC_Brigitte_LaCroix.json

```diff
@@ -28,7 +28,7 @@
       },
       { "skill": "survival", "bonus": { "rng": [ 2, 4 ] } },
       { "skill": "dodge", "bonus": { "rng": [ 4, 8 ] } },
-      { "skill": "melee", "bonus": { "rng": [ 4, 6 ] } },
+      { "skill": "melee", "bonus": { "rng": [ 6, 8 ] } },
```

Comment:
Why the skill increase? She's already one of the best companion NPC does she really need even higher stats?

Reply:
Hey, I could swear I was actually bumping her down, but I've been sort of tinkering with all of this stuff for months now, plus a bunch of stuff is actually getting merged *into* my changes so it's a bit messy. For sure I think 4-6 is good.